styles/styles.css

```diff
@@ -551,6 +613,23 @@ main .section.with-image {
 
 /* section background image styles */
 
+main div.section.hard-drive-background {
+  background-size: contain;
+  background-position-x: 50%;
+  background-position-y: 100%;
+  background-image: url("/icons/media_18801dc9f2feef2abd2efd2bf6dfacd12e4469a20.png");
```

Comment:
I'm not sure if this is the right way to use a background image. Adding a URL like **'media_18801dc9f2feef2abd2efd2bf6dfacd12e4469a20.png'** is neither author-friendly nor developer-friendly.
Why we are not adding this in the document itself ?